pkg/config/config.go

```diff
@@ -78,6 +85,33 @@ func LoadApiConfig() (*AppConfig, error) {
 		return nil, fmt.Errorf("environment variable SQS_QUEUE_NAME is not set")
 	}
 
+	sqsBatchSize := getEnv("SQS_BATCH_SIZE", "")
+	if sqsBatchSize == "" {
+		return nil, fmt.Errorf("environment variable SQS_BATCH_SIZE is not set")
```

Comment:
Instead of returning an error and fail to start, wouldn't be better to configure some defaults for these new settings if not explicitly configured via env vars?